Formula/crystal.rb

```diff
@@ -39,7 +39,7 @@ class Crystal < Formula
   depends_on "gmp" # std uses it but it's not linked
   depends_on "libevent"
   depends_on "libyaml"
-  depends_on "llvm@11"
+  depends_on "llvm@13"
```

Comment:
```suggestion
  depends_on "llvm"
```

Reply:
we prefer to stick the llvm version, as some versions are problematic

Reply:
we know llvm 13 is good for Crystal, but we had already troubles with llvm 11 that weren't fixed in 12, so we don't want to leave it open to the llvm gods.

Reply:
It's not possible to pin the version to `llvm@13` -- CI will just refuse to run because of the `tap_syntax` error.

Reply:
Sorry to take your time, but can you please lend me a hand here? Before pinning a specific version of llvm worked (as shown in the diff). Locally pinning the `llvm@13` works. So what's the issue here? Pinning is no longer accepted?

Reply:
`llvm@13` is an alias for the `llvm` formula. `llvm@11` works because that's not an alias. `brew` can resolve `llvm@13` just fine right now, but that can't/shouldn't be used as a dependency because there won't always necessarily be an `llvm@13` (e.g. when `llvm` is bumped to LLVM 14).

I suggest just changing this to `depends_on "llvm"` for now and we can fix it if LLVM 14 comes along and breaks things.